.travis.yml

```diff
@@ -26,8 +28,8 @@ before_install:
   - sudo apt-get update -y && sudo apt-get install -y libattr1-dev libblkid-dev linux-headers-$(uname -r) tree uuid-dev
   - mkdir -p $HOME/zfs
   - cd $HOME/zfs
-  - [[ -d spl-$rel.tar.gz ]] || curl -L https://github.com/zfsonlinux/zfs/releases/download/zfs-$rel/spl-$rel.tar.gz | tar xz
-  - [[ -d zfs-$rel.tar.gz ]] || curl -L https://github.com/zfsonlinux/zfs/releases/download/zfs-$rel/zfs-$rel.tar.gz | tar xz
+  - [[ -d spl-$rel.tar.gz ]] || curl -L https://github.com/openzfs/zfs/releases/download/zfs-$rel/spl-$rel.tar.gz | tar xz
```

Comment:
Spl was removed. It was merged into zfs. If you drop `0.6.5.11` than it is not needed anymore.

Reply:
Yes, might be good to do that; also looks like travis is failing on this line (invalid yaml?), so it'd be killing two birds with one stone

Reply:
oh, actually, it's still needed for v0.7 looks like

Reply:
It might be time to kill v0.7 though.

Reply:
Let me try spending some time on this later on; I tried changing but things failed (got travis working on my branch); probably made some mistake. Thanks!